fs2-aws-s3/src/test/scala/fs2/aws/s3/S3Suite.scala

```diff
@@ -198,6 +212,42 @@ class S3Suite extends CatsEffectSuite {
     }
   }
 
+  test("it should validate a valid eTag should validate successfully") {
```

Comment:
do you mind adding the test against the localstack, I hope it supports checksums

Reply:
@semenodm I thought the rest of the tests in the suite use local tack and I had used a real validator when calling uploadFileMultipart above.  Is there more that you would like?

Reply:
@semenodm I moved this to another class in 9b9fee3, let me know if you still need me to add additional tests.